src/package.xml

```diff
@@ -112,7 +112,6 @@
         <members>OPP_OpportunityNaming</members>
         <members>OPP_OpportunityNamingBTN_CTRL</members>
         <members>OPP_OpportunityNaming_BATCH</members>
-        <members>OPP_OpportunityNaming_TDTM</members>
```

Comment:
`OPP_OpportunityNaming_TDTM` class is still referenced in `STG_InstallScript_TEST` and `TDTM_DefaultConfig`.

In `STG_InstallScript_TEST` it is used as part of a test record that is inserted into the TDTM table.  I assume this is okay, and the value used in this test row doesn't matter?

In `TDTM_DefaultConfig` this class is referenced as a deleted class.  Does this reference remain even though the class itself is being removed from the package?  At what point does that reference get removed, if ever?